2001-3000/LIT2444Temher.xml

```diff
@@ -58,7 +58,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <p>Not to be confused with the text also known as <ref type="work" corresp="LIT3978Elmastoagya"/>
```

Comment:
You need something here to clearly disambiguate from the new general record.

Reply:
ok